### DISCUSSION TITLE
PHP SDK issue

### DISCUSSION THREAD

**Discussion by harishksoft** [kreait/firebase-php#521] [Q&A]
@jeromegamez I have installed the latest kreait using composer and try to call the below methods. but it says the function is not found. I am trying to integrate auth in my Codeigniter REST project.

// with an access token from Facebook
$signInResult = $auth->signInWithFacebookAccessToken($accessToken);

// with an ID token from Google
$signInResult = $auth->signInWithGoogleIdToken($idToken);

// with a Twitter OAuth 1.0 credential
$signInResult = $auth->signInWithTwitterOauthCredential($accessToken, $oauthTokenSecret);

**Comment by jeromegamez** [comment]
This is unfortunately not enough information to be able to help you. "latest" is not a version number, and the code is just copied from the documentation.

Please have a look at https://github.com/tattersoftware/codeigniter4-firebase for an integration with CodeIgniter. The creator of the integration is present on our Discord server, they might be able to help you further.

Please explain further what you're trying to do and what you're expecting to happen 🙏
